docs/user-guides/sharding.md

```diff
@@ -18,12 +19,14 @@ By default, Authorino will watch events related to all `AuthConfig` custom resou
 </table>
 
 <details markdown="1">
-  <summary>
+  <summary markdown="1">
     <strong>Authorino capabilities featured in this guide:</strong>
     <ul>
-      <li><a href="../architecture.md#sharding">Sharding</a></li>
-      <li>Identity verification & authentication → <a href="../features.md#api-key-authenticationapikey">API key</a></li>
+      <li>[Sharding](../architecture.md#sharding)</li>
+      <li>Identity verification & authentication → [API key](../features.md#api-key-authenticationapikey)</li>
```

Comment:
It seems to have broken in GitHub now.

<img width="1002" alt="Screenshot 2023-11-10 at 15 43 01" src="https://github.com/Kuadrant/authorino/assets/1842261/33ba2fe2-9bb3-4ca5-a1c0-884775db4aba">

Reply:
Sadly the only way I could get this to render correctly in both place was by using HTML list with A tags and absolute paths to page on the website.

Reply:
How is it that it works for all other user guides then? Or maybe it doesn't. In this case, why changing this particular file and not all the others?